backend/src/infra/network/outbound-url-policy.ts

```
@@ -0,0 +1,221 @@
+  for (let index = 0; remaining > 0; index++) {
+    const width = Math.min(remaining, 16);
+    const mask = width === 16 ? 0xffff : 0xffff << (16 - width);
+    if ((groups[index] & mask) !== (prefix[index] & mask)) return false;
+    remaining -= width;
+  }
+  return true;
+}
+
+function isPrivateIpv6(address: string): boolean {
+  const groups = parseIpv6(address);
+  if (!groups) return false;
```

ESLint

[error] Expected { after 'if' condition.

(curly)